homeassistant/components/oncue/sensor.py

```diff
@@ -0,0 +1,164 @@
+"""Support for Oncue sensors."""
+from __future__ import annotations
+
+from aiooncue import OncueDevice, OncueSensor
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ELECTRIC_POTENTIAL_VOLT,
+    FREQUENCY_HERTZ,
+    PERCENTAGE,
+    POWER_WATT,
+    PRESSURE_PSI,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
```

Comment:
Would have been nice if entity categories have been specified in the descriptions.

Reply:
https://github.com/home-assistant/core/pull/63231